### PR TITLE
FT2021-2RC1

### PR DESCRIPTION
**Pull request by blchoy**
I am pleased to create this pull request which signals completion of the preparation of IWXXM Version 2021-2RC1.  Subject to the review of at least one member of the team, could the Secretariat please merge this branch with the master branch, publish a new release and transfer the files to schemas.wmo.int for public consultation?

I shall clean up the other stuff on the repository (issues, project, etc.) afterwards.

### CODE REVIEW

File: IWXXM/common.xsd [hunk 1]
```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/3.0" version="3.0-dev" xmlns:iwxxm="http://icao.int/iwxxm/3.0" xmlns:aixm="http://www.aixm.aero/schema/5.1.1" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns="http://www.w3.org/2001/XMLSchema">
+<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/2021-2" version="3.1.0RC1" xmlns:iwxxm="http://icao.int/iwxxm/2021-2" xmlns:aixm="http://www.aixm.aero/schema/5.1.1" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns="http://www.w3.org/2001/XMLSchema">
+	<include schemaLocation="iwxxm.xsd"></include>
```
[the remark this answers is not in the document]
Comment (amilan17) [reply IWXXM/common.xsd:3]: @mgoberfield -- I tested the XSDs in Oxygen and this did not break anything for me. Is this causing problems for you?

**Review by amilan17 — APPROVED** [review]
I reviewed the XSDs, XMLs and I tested basic functionality in Oxygen. I'm ready proceed with PR, pending the resolution of Mark's suggestion.

File: IWXXM/common.xsd (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgoberfield) [reply IWXXM/common.xsd:3]: It is just a concern that some software may not handle such loops in a graceful manner such as oxygen does.

**Comment by blchoy** [comment]
I have committed the changes as requested.  I have also corrected iwxxm-collect.xsd which was still pointing to IWXXM 3.0.  You can see the changes at https://github.com/wmo-im/iwxxm/pull/259/commits/8383493c84c63fad342b0f3fbc24070cf7394160

**Pull request merged**